script/foreman-admin

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env ruby
+# -*- encoding: utf-8 -*-
+
+require 'rubygems'
+require 'clamp'
+
+module ForemanAdmin
+  class Command < ::Clamp::Command
+    def self.command_name(name = nil)
+      @command_name = name if name
+      @command_name
+    end
+
+    def command_name
+      self.class.command_name
+    end
+
+    def self.description(description = nil)
+      @description = description if description
+      @description
+    end
+
+    def description
+      self.class.description
+    end
+
+    def execute
+      # override me
+    end
+  end
+
+  class ExternalCommand < Command
+    # The base command that will be executed externally.
+    # Think of this as ARGV[0] of what we're running externally.
+    def self.external_invocation(invocation = nil)
+      @external_invocation = invocation if invocation
+      @external_invocation
+    end
+
+    def external_invocation
+      self.class.external_invocation
+    end
+
+    # The exact assembled command that will be executed externally.
+    # Override this in a child class if you want it have logic.
+    def external_command
+      "#{external_invocation}"
+    end
+
+    def execute
+      puts `#{external_command}`
+    end
+  end
+
+  class BackupCommand < ForemanAdmin::ExternalCommand
+    command_name 'backup'
+    description 'Backup your Foreman server'
+    external_invocation '/usr/sbin/foreman-backup'
```

Comment:
No, this is not currently a thing. There are PRs open for a backup script as well as a restore script for a foreman/katello server. I've commented out the loading of `BackupCommand` because the executable is not available yet.